common/ssl_calls.c

```diff
@@ -485,6 +486,60 @@ ssl_gen_key_xrdp1(int key_size_in_bits, const char *exp, int exp_len,
     return error;
 }
 
+/*****************************************************************************/
+/** static DH parameter, can be used if no custom parameter is specified
+see also 
+ * https://wiki.openssl.org/index.php/Diffie-Hellman_parameters
+ * https://wiki.openssl.org/index.php/Manual:SSL_CTX_set_tmp_dh_callback(3)
+*/
```

Comment:
This part doesn't meet our coding standard but I know this is autogenerated code by OpenSSL so it's OK with this.